modules/custom/az_paragraphs/src/Plugin/migrate/process/ParagraphsBehavior.php

```diff
@@ -9,8 +9,14 @@
 /**
  * Configure Behavior for paragraphs.
  *
- * @MigrateProcessPlugin(
- *   id = "paragraphs_behavior_settings"
+ * @deprecated in az_quickstart:2.1.3 and is removed from az_quickstart:2.3.0.
+ *   Use the
+ *   \Drupal\az_paragraphs\Plugin\migrate\process\AZParagraphsBehaviorSettings
+ *   process plugin instead following its migration patterns.
```

Comment:
Maybe we could just use the ID in this comment?
```suggestion
 *
 * Use the az_paragraphs_behavior_settings process plugin instead following its
 * migration patterns.
```